src/ui/map.js

```diff
@@ -1215,11 +1215,9 @@ class Map extends Camera {
                     this.style._sourceCaches[id].clearTiles();
                 }
             }
-            this.style.applyProjectionUpdate();
             this._update(true);
```

Comment:
Was the order between `applyProjectionUpdate` and `_update` important? I notice that `_update` eventually triggers a new frame. So I wonder if this update could, as a side effect, delay the projection update by one frame.

Reply:
Good catch Karim! I've tried switching the order and am seeing an improvement to symbol rendering speed.